chart/templates/deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         - /manager
         env:
         - name: FLAGD_VERSION
-          value: INPUT_FLAGD_VERSION
+          value: latest
```

Comment:
Is this change intended

There seems to be a shell script for `sed`ing this in `hack/`.

Reply:
Please don't do this, that value is set intentionally